src/pages/dev/general-message-passing/express.mdx

```diff
@@ -76,11 +83,22 @@ await api.estimateGasFee(
 
 ## Risks
 
+Using the Axelar Express Service comes with risks. Please keep these in mind when integrating your contracts.
+
 ### Funding
-Using Axelar Express comes with risks. Namely if anything changes on the source chain from the time the transaction is sent to the time it is finalized (due to a [block reorg](https://cointelegraph.com/explained/what-is-chain-reorganization-in-blockchain-technology) for example) and the transaction on the destination chain has already executed then that could lead to lost funds. These funds would be lost as the transaction will have been instantly executed on the destination chain on the assumption of logic that never took place on the source chain. Due to the potential risk here, liquidity must be provided for the Express Relayers by a party to assume the risk that the relayer is taking when it loans out funds on the destination chain.
 
-### Execution
-On the destination chain an Axelar relayer triggers the `_execute()`/ `_executeWithToken()` function to trigger a transaction. The GMP payload being used can be assumed to be true as it passes through the [validation process](https://github.com/axelarnetwork/axelar-cgp-solidity/blob/4d01eb6ea1b5413520d9ecab804f815103021148/contracts/AxelarGateway.sol#L233) on the Axelar Gateway. In an express call however the execution on the destination chain occurs before the source chain finalizes, meaning that you cannot rely on this validation process. To mitigate, data that is being passed in you can make use of the unique [`CommandId`](https://github.com/axelarnetwork/axelar-core/blob/main/docs/cli/axelard_query_evm_command.md). The `CommandId` is a unique identifier, which is available on the destination chain you're transacting to. With your unique commanId you can reconstruct the message that was passed in. 
+Since the Express service instantly executes the transaction on the destination chain, any changes on the source chain between the time the transaction is sent and the time it is finalized (for example, due to a [block reorg](https://cointelegraph.com/explained/what-is-chain-reorganization-in-blockchain-technology)) could lead to funds being lost.
+
+These funds would be lost because the transaction will have been instantly executed on the destination chain on the assumption of logic that never took place on the source chain.
+
+To mitigate this risk, the transacting party must provide liquidity to the Express relayers for all Express transactions. This liquidity will act as insurance against the risk the relayer takes when loaning out funds on the destination chain. The amount of funds should match the value of the transaction itself, and will be returned when the transaction has finalized on the source chain.
```

Comment:
does not have to be the transacting party necessarily. Could be anyone, even us. Just someone has to supply the collateral to assume the risk (though typically it would be the team that wants to onboard onto express but doesn't have to be)